### ISSUE TITLE
bug(a11y): action elements with role link have screen reader problems

### ISSUE THREAD

**Issue by jeripeierSBB** [sbb-design-systems/lyne-components#1868]
- Firefox + NVDA reads "link link"
 - Chrome + JAWS doesn't read "link" at all

idea:
 - using other role than "presentation" on inner a tag

**Comment by jeripeierSBB** [comment]
discussed with A4A, awaiting feedback.
